packages/core/src/types/module.ts

```diff
@@ -344,7 +344,7 @@ export type SolidityParameterType =
  *
  * @beta
  */
-export type ModuleParameterType = SolidityParameterType;
+export type ModuleParameterType = SolidityParameterType | AccountRuntimeValue;
```

Comment:
What do you think of leaving `ModuleParameterType` unchanged but altering just default value to be `ModuleParameterType | AccountRuntimeValue | undefined` or whatever?

Are there any implications to expanding the `ModuleParameterType` to include `AccountRuntimeValue` outside of default value, or in the future?

Reply:
Hm, that's an interesting idea I hadn't thought about. In all honesty, I'm not sure what the implications might be.

Reply:
Maybe one way to think about this is  are there any places the current change would allow us to use `AccountRuntimeValue` that are unintended?
I suspect changing the default value will lead to more code changes but less "conceptual  " changes.

Reply:
I've added a change in https://github.com/NomicFoundation/hardhat-ignition/pull/679/commits/9e480042f745a5c0e86d49e5191ed91976e7f9b5 that fixes the one place I was able to find that was impacted by changing `ModuleParameterType` as opposed to changing the default value type. I believe I've gone over anywhere relevant and this was the only place that would have been impacted.